src/utils/TextToSpeech.js

```diff
@@ -0,0 +1,74 @@
+const supportedVoiceLanguages = ['en', 'it'];
+let recommendedVoices = [];
```

Comment:
nit: I think you can use `const` over `let` here?

Reply:
I often get mixed up on what qualifies as an assignment for arrays in JS, though, so could be wrong!

Reply:
Also, should this be an object over an array? (sorry for the extensive comments at this point)

Reply:
Oh, good calls.  Fixed.

Reply:
...is this fixed?

Reply:
Thanks for catching this; somehow the fix didn't get committed.  Should be updated now.